app/posts/digital-public-works-ideas.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Ideas for digital public works
+description: Some thoughts on digital platforms and services that would be helpful for cross-government teams.
+date: 2024-03-01
+author:
+  name: Frankie Roberto
+  url: https://github.com/frankieroberto
+---
+
+A recent blog post from dxw suggests that [there needs to be more support for digital public works](https://www.dxw.com/2024/01/there-needs-to-be-more-support-for-digital-public-works/), described as platforms and tools which work across government to make it easier and faster to build services.
+
+I’m also big supporter of these. This unofficial X-GOVUK community is one way to support cross-government collaboration and tools, but many digital public works would need full time teams to support them.
+
+There’s lots to be thankful for. The [platforms developed by the Government Digital Service](https://www.gov.uk/service-toolkit#platforms-and-tools) – Notify, Pay, One Login and Forms – are all super useful.
+
+Unfortunately there doesn’t currently seem to be an easy way for government departments working on services to spot other opportunities for common platforms and to turn these into funded cross-government teams and services.
```

Comment:
I feel it's not so much the spotting, as the highlighting / coordinating. We spot opportunities all the time!

Reply:
Agreed - is it an easy way to suggest them?

Reply:
I don’t think so? (Which is half the point of this post). I think traditionally only GDS have ever built cross-government digital services, and I don’t know if there are any mechanisms for departments to lobby GDS to create new ones?

Perhaps there could also be ways for departments themselves to band together to create cross-department teams to investigate ideas for services?